### PR TITLE
Remove usage of Spring Boot's `LiveReloadServer`

### LINKED ISSUE #277: Gretty 3.0.8+ is incompatible with Proguard `7.3.0`
`build.gradle`:
```gradle
plugins {
	id 'org.gretty' version '3.0.8' apply false
}
```

`settings.gradle`:
```gradle
include 'foo'
```

`foo/build.gradle`:
```gradle
import proguard.gradle.ProGuardTask

buildscript {
	repositories {
		mavenCentral()
	}

	dependencies {
		classpath 'com.guardsquare:proguard-gradle:7.3.0'
	}
}

tasks.register('proguardFoo', ProGuardTask)
```

Running `gradle foo:proguardFoo` (with Gradle 7.5.1) on this fails with:

```
> Task :foo:proguardFoo FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':foo:proguardFoo'.
> Could not initialize class org.apache.logging.log4j.util.PropertiesUtil
```

Downgrading `ProGuard` to `7.2.2` or downgrading Gretty to `3.0.7` fixes the issue (i.e. there is another exception `Index 0 out of bounds for length 0` which is normal however).

Initially I thought that this was because of the recent changes to classloader filtering (as I'm using Gretty 3.1.0 and was testing with it). However, the fact that Gretty 3.0.8 also doesn't work... is extremely strange. Also note that Gretty is not even applied here but without this line, it works correctly.

@f4lco, any ideas?

### PR DESCRIPTION
**Pull request by boris-petrov**
Fixes #277

cc @f4lco

### CODE REVIEW

File: libs/gretty/src/main/groovy/org/akhikhl/gretty/StartBaseTask.groovy [hunk 1]
```diff
@@ -54,10 +52,6 @@ abstract class StartBaseTask extends DefaultTask {
   void action() {
     LauncherConfig config = getLauncherConfig()
     Launcher launcher = new DefaultLauncher(project, config)
-    if(config.serverConfig.liveReloadEnabled) {
-      launcher.optionalLiveReloadServer = new OptionalLiveReloadServer(new LiveReloadServer())
-    }
-    launcher.scannerManager = createScannerManager(config, launcher.optionalLiveReloadServer)
```
Comment (f4lco) [inline libs/gretty/src/main/groovy/org/akhikhl/gretty/StartBaseTask.groovy:60]: This is removing too much, is it?
This patch would kill the only usage of `JDKScannerManager` and `JettyScannerManager`, and probably remove any redeploy-on-file-change functionality of Gretty, right? (I did not test it).
I suppose we would need to leave the call to `createScannerManager` in, and just remove the `OptionalLiveReloadServer` bits.

**Review by f4lco — CHANGES_REQUESTED** [review]
I'm fine with the removal of the live reload server, just get back to the comment on (possibly) removing all reload / redeploy functionality from Gretty 😄

File: libs/gretty/src/main/groovy/org/akhikhl/gretty/StartBaseTask.groovy (hunk 1 above)
Comment (boris-petrov) [reply libs/gretty/src/main/groovy/org/akhikhl/gretty/StartBaseTask.groovy:60]: You're right! I did what you suggested, please take a look. :)

**Review by f4lco — APPROVED** [review]
That's it, great 🏅

**Pull request merged**